### PR TITLE
Display celebratory confetti on solve

### PR DESCRIPTION
**Pull request by cleercode**
Displays confetti for 7 seconds whenever the game goes from being unsolved to solved. Doesn't show on mobile to avoid performance issues.

Just an idea. Feel free to take or leave. :)

image

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -11842,6 +11842,13 @@ react-app-polyfill@^1.0.6:
     regenerator-runtime "^0.13.3"
     whatwg-fetch "^3.0.0"
 
+react-confetti@^5.0.1:
+  version "5.0.1"
+  resolved "https://unpm.uberinternal.com/react-confetti/-/react-confetti-5.0.1.tgz#3003570778cafad4db0ab5513018d4c10e9d1271"
```
Comment (stevenhao) [inline yarn.lock:11847]: oops :P

Comment (cleercode) [reply yarn.lock:11847]: lol 😅 will fix

**Comment by TMiguelT** [comment]
Greatest feature yet

**Comment by catherynl** [comment]
Omg love this

**Pull request merged**